docs/iris/src/_templates/index.html

```diff
@@ -13,15 +13,6 @@
 #sliderNav div { display: inline-block; height: 8px; width: 8px; margin: 0 2px;
   border: #ccc medium solid; border-radius: 50%; box-shadow: 0 0 2px 0 #ccc;}
 #sliderNav div.activeSlide { border-color: #777; background-color: #ccc; }
-
-
-.bodywrapper {
```

Comment:
This was removed, as I added it to hide the search bar. I think that is premature for the release.
Long-term, I'd like to move the search to the nav bar at the top.